webroot/results/admin/persons_check_finished.php

```diff
@@ -183,20 +183,20 @@ function checkTooMuchInResults () {
 #----------------------------------------------------------------------
   global $personsFromPersons, $personsFromResults;
   echo "<hr />";
-  
+
   $tooMuchInResults = array();
   #--- Find all ('finished') entries in Results that don't have a match in Persons.
   foreach( array_keys( $personsFromResults ) as $personKey ){
-    if( $personsFromResults[$personKey]['id']  &&  ! isset($personsFromPersons[$personKey]) )
+    if( $personsFromResults[$personKey]['id'] && !ctype_digit($personsFromResults[$personKey]['id']) &&  ! isset($personsFromPersons[$personKey]) )
```

Comment:
It took me a while to figure out what's going on here. Could you add a comment here explaining why person ids can be all numeric now?